dist/profile/manifests/jira.pp

```diff
@@ -0,0 +1,68 @@
+# Run containerized JIRA to serve issues.jenkins-ci.org
```

Comment:
Will you update the comment to include a link to the docker container repo, which I'm hoping has a good readme on how the container is all built out and whatnot?